packages/bot-web-ui/src/components/dashboard/dashboard-component/__tests__/run-strategy.spec.tsx

```diff
@@ -0,0 +1,31 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import RunStrategy from '../run-strategy';
+
+jest.mock('../../../trade-animation/trade-animation', () =>
```

Comment:
@maryia-matskevich-deriv do we have to mock this component or we can just let the component render itself? If we don't use the mock then maybe we also need the Store provider and Dbot store provider as a wrapper.

Reply:
@shafin-deriv I used the mock for the internal component because this test is to test the `RunStrategy`, so this is a good way for the unit test. 
If we had an integration test and we wanted to test the internal component `Trade animation` to the transfer of some properties, then it would be appropriate to don't use the mock,  then we need the Store provider and Dbot store provider as a wrapper for properties

Reply:
@shafin-deriv In the current case the properties are static, not from the store. But still, we should check static props with the help data-testid. Since they are there and if some developer breaks them, they will immediately notice and fix them. After all, this is the default behavior of this component
<img width="555" alt="Screenshot 2023-08-21 at 11 53 35" src="https://github.com/binary-com/deriv-app/assets/103181650/302751ea-904f-44c3-9fe9-ba40336a7900">

Reply:
I mean in this case, if someone removes those props from the real component it will not break the test and the test case will still pass as these props are being checked from the mock you create and they will always be there even if it is removed from the actual implementation.

Reply:
@shafin-deriv Yees, i totally agree with you, it would be better not to test the internal component here at all. We will test it in a separate file for all its cases anyway. I need to refactor the code for me

Reply:
✅ done